tests/datasource/fluent/_fake_cloud_api.py

```diff
@@ -98,7 +102,7 @@ class CloudDetails(NamedTuple):
     # using alternative syntax for creating type dict because of key names with hyphens
     # https://peps.python.org/pep-0589/#alternative-syntax
     {
-        "me": Dict[str, str],
+        "me": Dict[str, Any],
```

Comment:
The dict value is now more complicated depending on the key and is documented elsewhere (and can be inferred by looking at the example value in this file).